administration/intro.adoc

```diff
@@ -12,26 +12,13 @@ Requirements
 A few requirements need to be met before you can begin:
 
 * https://kubernetes.io[Kubernetes] cluster
-(https://github.com/openshift/origin[OpenShift], Tectonic)
+(https://github.com/openshift/origin[OpenShift], Tectonic) Based on Kubernetes 1.10 or greater
+* Kubernetes apiserver must have `--allow-privileged=true` in order to run KubeVirt's privileged DaemonSet.
 * `kubectl` client utility
 * `git`
```

Comment:
Why do we need git?

Reply:
we don't... that's such a weird thing to call out as a requirement to use kubevirt.